includes/emails/email-summary/class-edd-email-summary.php

```diff
@@ -0,0 +1,791 @@
+						width: 8px !important;
+						height: 6px !important;
+						vertical-align: baseline !important;
+					}
+					.table-top-title {
+						font-size: 12px !important;
+						line-height: 15px !important;
+					}
+					table.top-products tr th {
+						padding: 6px 0px !important;
+						font-size: 10px !important;
+						line-height: 13px !important;
+					}
+					table.top-products tr td {
+						font-size: 10px !important;
+						padding: 6px 0px !important;
+						border-bottom: 1px solid #F0F1F4;
+					}
+					table.top-products tr td:nth-child(2) {
+						font-size: 12px !important;
+					}
+					.pro-tip-holder {
+						background: #F3F8FE;
+						border-radius: 8px !important;
+						padding: 24px 24px !important;
+					}
+					.pro-tip-section-title {
+						font-size: 14px !important;
+						line-height: 15px !important;
+					}
+					.pro-tip-section-title img {
+						margin-right: 3px !important;
+					}
+					.pro-tip-title {
+						font-size: 12px !important;
+						line-height: 16px !important;
+					}
+					.cta-btn {
+						padding: 6px 15px;
+						font-weight: 600;
+						font-size: 14px;
+						line-height: 20px;
+						background: #2794DA;
+						display: inline-block;
+						color: #FFFFFF;
+						text-decoration: none;
+					}
+				}
+			</style>
+
+			<body style="margin: 0px;">
+				<!-- PREVIEW TEXT -->
+				<div style="display: none; max-height: 0px; overflow: hidden;">
+					<?php echo esc_html( __( 'Store performance summary', 'easy-digital-downloads' ) ); ?> <?php echo esc_html( $date_range['start_date']->format( $wp_date_format ) ); ?> - <?php echo esc_html( $date_range['end_date']->format( $wp_date_format ) ); ?>
+				</div>
+
+				<!-- HEADER HOLDER -->
+				<div class="email-header-holder" style="background: #343A40; max-height: 60px; height: 60px;">
+
+					<div class="email-container" style="max-width: 450px; margin: 0 auto; font-family: 'Source Sans Pro', sans-serif; color: #1F2937;">
+
+						<div class="logo-holder" style="padding: 12px 31px 7px 31px; display: inline-block;">
+							<img src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/edd-logo-white-2x.png' ); ?>" class="edd-logo" width="216" height="35" style="width: 100%; max-width: 100%; height: auto; max-width: 216px; max-height: 35px;">
+						</div>
+
+					</div>
+				</div>
+
+				<div class="email-container" style="max-width: 450px; margin: 0 auto; font-family: 'Source Sans Pro', sans-serif; color: #1F2937;">
+
+
+					<!-- MAIN EMAIL CONTENT HOLDER -->
+
+					<div class="content-box" style="background: #FFF;">
+
+						<div class="content-holder" style="padding: 22px 31px 0px 31px;">
+
+							<h1 style="margin: 0px; color: #1F2937;font-weight: 700;font-size: 24px;line-height: 24px;"><?php echo esc_html( __( 'Your eCommerce Summary', 'easy-digital-downloads' ) ); ?></h1>
+
+							<div class="period-date pull-down-8" style="margin-top: 8px; font-weight: 400; font-size: 14px; line-height: 18px; color: #4B5563;">
```

Comment:
Since it is in the email wrapper, they are necessary in order to maintain a consistent set of styles. Most email clients 'sandbox' the markup so we are just fine doing this.